src/UI/TransactionToast/index.tsx

```diff
@@ -29,16 +33,14 @@ const TransactionToast = ({
   toastId,
   title = '',
   shouldRenderDefaultCss = true,
-  className = 'transaction-toast',
-  withTxNonce = false,
+  className = '',
   transactions,
   status,
```

Comment:
Let's make this component a dumb component and move the logic to transactionToastComponent